src/sentry/static/sentry/app/views/organizationEvents/eventsChart.jsx

```diff
@@ -1,69 +1,157 @@
+import {pick, isDate, isEqual, isEqualWith} from 'lodash';
 import {withRouter} from 'react-router';
 import PropTypes from 'prop-types';
 import React from 'react';
 import moment from 'moment';
 
+import {getFormattedDate} from 'app/utils/dates';
 import {t} from 'app/locale';
+import DataZoom from 'app/components/charts/components/dataZoom';
 import LineChart from 'app/components/charts/lineChart';
 import EventsContext from 'app/views/organizationEvents/utils/eventsContext';
 import EventsRequest from 'app/views/organizationEvents/utils/eventsRequest';
 import SentryTypes from 'app/sentryTypes';
+import ToolBox from 'app/components/charts/components/toolBox';
 import withApi from 'app/utils/withApi';
 
 const DEFAULT_GET_CATEGORY = () => t('Events');
 
-class EventsChart extends React.PureComponent {
+const dateComparator = (value, other) => {
+  if (isDate(value) && isDate(other)) {
+    return +value === +other;
```

Comment:
Could we use something like `moment.isSame()` rather than rolling our own date comparison functions?

Reply:
The comparison itself is simple enough that it doesn't need `moment` imo and the actual `dateComparator` fn can't have `moment.isSame` as a drop in replacement since this is a comparator used for lodash's `isEqual` check. We only do the date comparison if both are dates.